### PR TITLE
Add Satellite Service Config

### LINKED ISSUE #37: Satellite consumer app
## Detailed Description
Create satellite app that runs in ecs every 5 mins
create iam and roles

## Possible Implementation
Copy the NWP consumer, just change docker image and

### PR DESCRIPTION
**Pull request by jacobbieker**
# Pull Request

## Description

Adds the satellite consumer service, based off the NWP one

Fixes #37 

## How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration_

- [ ] Yes

_If your changes affect data processing, have you plotted any changes? i.e. have you done a quick sanity check?_

- [ ] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: terraform/modules/services/sat/secrets.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Read in secrets for API for EUMETSAT API
+
+data "aws_secretsmanager_secret" "sat-api" {
+  name = "development/consumer/sat"
+  #  arn = "arn:aws:secretsmanager:eu-west-2::secret:development/consumer/nwp"
```
Comment (peterdudfield) [inline terraform/modules/services/sat/secrets.tf:5]: remove if not needed

**Comment by peterdudfield** [comment]
Hey Jacob, 

great work on this, well done for doing this! 

Have you tried deploying this on the development environment? and did it work>? i.e did some files end up in s3? 
If not perhaps we could do it together sometime this week? If it works, then we can merge it in. What do you think?

**Comment by jacobbieker** [comment]
> Hey Jacob,
> 
> great work on this, well done for doing this!
> 
> Have you tried deploying this on the development environment? and did it work>? i.e did some files end up in s3? If not perhaps we could do it together sometime this week? If it works, then we can merge it in. What do you think?

Hi, I haven't yet, yeah, sounds great to try deploying it together this week!

File: terraform/modules/services/sat/secrets.tf (hunk 1 above)
Comment (jacobbieker) [reply terraform/modules/services/sat/secrets.tf:5]: This should be needed actually, I think, as I need to read in the EUMETSAT API keys

File: terraform/modules/services/sat/iam.tf [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_iam_role" "consumer-nwp-iam-role" {
```
Comment (peterdudfield) [inline terraform/modules/services/sat/iam.tf:96]: change to sat

File: terraform/modules/services/sat/schedule.tf [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# This is an temporary module.
+# This will schedule the aws task to run on cron job.
+# We want to move to Dagster but for the moment its useful to have this setup
+
+resource "aws_cloudwatch_event_rule" "event_rule" {
+  name                = "sat-schedule-${var.environment}"
+  schedule_expression = "cron(*/5 * * * *)"
+  # runs every day 8.00, 14.00 and 20.00, will have to update this for production
```
Comment (peterdudfield) [inline terraform/modules/services/sat/schedule.tf:8]: update

File: terraform/modules/services/sat/secrets.tf (hunk 1 above)
Comment (peterdudfield) [reply terraform/modules/services/sat/secrets.tf]: sorry i meant the comment  `# arn`

**Pull request merged**